.gitignore

```diff
@@ -1,5 +1,4 @@
 node_modules/
 dist/
 benchmarks/results/
-package-lock.json
```

Comment:
Please remove package-lock.json

Reply:
yes will do. I've only ever developed applications so I listen to you 👍